openassessment/xblock/ui_mixins/mfe/page_context_serializer.py

```diff
@@ -294,7 +294,7 @@ def get_submission(self, instance):
             if active_step == "training":
                 response = instance.student_training_data.example
             elif active_step == "peer":
-                response = instance.peer_assessment_data().get_peer_submission()
+                response = instance.peer_assessment_data().get_active_assessment_submission()
```

Comment:
This looks to me like we will have to separately trigger a `get_peer_submission` when we load the page if we don't have a submission yet... I would think we want to `get_new_or_active_assessment_submission`, no?

@muselesscreator, correct me if I'm wrong.

Reply:
This is what I was told yesterday but I have better clarity on the desired behavior now.